### PR TITLE
Feature/documents page

### PR DESCRIPTION
**Pull request by mahid797**
### _No Documents Empty State_
image

### _With Documents & Pagination (Pagination Autohides when not enough items)_
image

`Issue #5 Documents Page
`

- Might require some changes when layout and sidebar are finalized. 
- Haven't yet connected to backend
  - Drag & Drop works but still needs complete functionality
  - Fetching Documents
  - Functionality for the Menu remains

### CODE REVIEW

**Review by Paxxack — COMMENTED** [review]
Changes required for PR#27

File: Client/src/app/team/components/FilterToggle.tsx
[the remark this answers is not in the document]
Comment (mahid797) [reply Client/src/app/team/components/FilterToggle.tsx:1]: This is a issue due to previous PRs not being merged with Master. Now that they are, they wont show up.

File: Client/src/components/Dropdown.tsx
[the remark this answers is not in the document]
Comment (mahid797) [reply Client/src/components/Dropdown.tsx:1]: This is an issue due to previous PRs not being merged with Master. Now that they are merged, these files won't show up.

File: Client/src/theme/theme.ts
[the remark this answers is not in the document]
Comment (mahid797) [reply Client/src/theme/theme.ts:1]: Done

File: Client/public/docBackground.svg
[the remark this answers is not in the document]
Comment (mahid797) [reply Client/public/docBackground.svg:1]: All of these icons are placeholders for actual icons. Once we decide which ones and how to use them (server or package), these icons would be removed.

File: Client/src/app/documents/components/DragAndDropBox.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			/>
+
+			<Typography variant="body1" fontWeight={600}>
+				{text}
+			</Typography>
+			<input
+				type="file"
+				id="file-input"
+				style={{ display: 'none' }}
+				onChange={(e) => {
```
[the remark this answers is not in the document]
Comment (mahid797) [reply Client/src/app/documents/components/DragAndDropBox.tsx:42]: These are just placeholders until we have a backend

File: Client/src/app/documents/components/DocumentsTableHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+export const DocumentsTableHeader = ({
+	orderBy,
+	orderDirection,
+	onSort,
+}: Props) => (
+	<TableRow>
+		<TableCell sx={{ width: '5%' }}></TableCell>
+		<TableCell
+			sx={{
+				color: 'rgba(161, 175, 198)',
```
Comment (neerajsunil) [inline Client/src/app/documents/components/DocumentsTableHeader.tsx:21]: these can be colors added to the theme

File: Client/src/app/documents/components/DocumentsTableHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+				width: '45%',
+			}}>
+			DOCUMENT
+		</TableCell>
+		<TableCell
+			sx={{
+				color: 'rgba(161, 175, 198)',
```
Comment (neerajsunil) [inline Client/src/app/documents/components/DocumentsTableHeader.tsx:28]: theme colors

**Comment by neerajsunil** [comment]
The pagination has two button that does the same thing, we should remove the one beside the number.
image

File: Client/src/app/documents/components/DocumentsTableHeader.tsx (hunk 2 above)
Comment (mahid797) [reply Client/src/app/documents/components/DocumentsTableHeader.tsx:21]: Will add them in the refactor PR(Issue #31)

File: Client/src/app/documents/components/DocumentsTableHeader.tsx (hunk 3 above)
Comment (mahid797) [reply Client/src/app/documents/components/DocumentsTableHeader.tsx:28]: Will add them in the refactor PR(Issue #31)

**Comment by mahid797** [comment]
> The pagination has two button that does the same thing, we should remove the one beside the number. image

This is also a theme integration issue. Should be resolved in the refactor

**Review by Paxxack — APPROVED** [review]
There is few things we are going to talk about but the code for my part seems good to go. Good job!

**Pull request merged**